### PR TITLE
added check for the existence of modules imagick, xdebug

### PR DESCRIPTION
**Pull request by odebretseni**

### CODE REVIEW

File: tasks/configure-php-settings.yml [hunk 1]
```diff
@@ -54,13 +54,24 @@
     PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
   changed_when: false
 
+- name: Get imagick module exist status
+  stat:
+    path: /usr/local/lib/php/pecl/20160303/imagick.so
```
Comment (BrunIF) [inline tasks/configure-php-settings.yml]: maybe try use variable? `20160303`

File: tasks/configure-php-settings.yml [hunk 2]
```diff
@@ -54,13 +54,24 @@
...
   args:
     executable: /usr/local/opt/php@7.1/bin/pecl
   environment:
     PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
   changed_when: false
+  when: not imagick.stat.exists
+
+- name: Get imagick module exist xdebug
+  stat:
+    path: /usr/local/lib/php/pecl/20160303/xdebug.so
```
Comment (BrunIF) [inline tasks/configure-php-settings.yml]: also?

File: tasks/configure-php-settings.yml [hunk 3]
```diff
@@ -69,6 +80,7 @@
   environment:
     PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```
Comment (BrunIF) [inline tasks/configure-php-settings.yml]: Try to use `args` and run from PECL directory

File: tasks/deploy-dashboard.yml [hunk 4]
```diff
@@ -1,8 +1,11 @@
 ---
 - name: Run deploy dashboard script
   command: /Users/{{ work_user }}/{{ work_dir }}/{{ dashboard_dir }}/sh/update.sh
-  args:
-    executable: /bin/zsh
   environment:
     PATH: "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
```
Comment (BrunIF) [inline tasks/deploy-dashboard.yml]: `args` run with bash

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: tasks/deploy-dashboard.yml (hunk 4 above)
Comment (odebretseni) [reply tasks/deploy-dashboard.yml]: [WARNING]: As of Ansible 2.4, the parameter 'executable' is no longer supported with the 'command' module. Not using '/bin/bash'.

File: templates/horizon.ini.j2 [hunk 5]
```diff
@@ -0,0 +1,8 @@
+[program:horizon]
+process_name=%(program_name)s_%(process_num)02d
+command=/usr/local/opt/php@7.1/bin/php /Users/{{ work_user }}/{{ work_dir }}/{{ dashboard_dir }}/artisan horizon
```
Comment (BrunIF) [inline templates/horizon.ini.j2]: if will use 7.4 PHP ?

File: tasks/configure-php-settings.yml [hunk 6]
```diff
@@ -69,6 +78,7 @@
...
   changed_when: false
+  when: not imagick.stat.exists
 
 - name: Start service php
   command: brew services start php@7.1
```
Comment (BrunIF) [inline tasks/configure-php-settings.yml]: Version in defaults

**Review by BrunIF — CHANGES_REQUESTED** [review]

File: templates/info.txt.j2 [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  URL Socket site       - https://home.{{ work_domain }}:9090
+  URL Mailcatcher site  - http://127.0.0.1:1080
+  URL Supervisor site   - http://127.0.0.1:9001
+
+ If you want to pull the latest changes, switch to the necessary branch and run the script,
+which can be found on the following path - /Users/{{ work_user }}/{{ work_dir }}/{{ dashboard_dir }}/sh/update.sh
+
+ If you want to run basic tests, run the following from the project folder:
+
+ - npm run test //carma-test
```
Comment (BrunIF) [inline templates/info.txt.j2]: Change like a Promo

**Review by BrunIF — CHANGES_REQUESTED** [review]

**Review by BrunIF — APPROVED** [review]

**Pull request merged**